mobile/src/libs/components/card/card.tsx

```diff
@@ -6,52 +6,65 @@ import {
   Icon,
   Image,
   Pressable,
+  Swipeable,
   Text,
   View,
 } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
 import { type IconName } from '#libs/types/types';
 
 import { DEFAULT_NUMBER_OF_LINES } from './libs/constants/constants';
 import { styles } from './styles';
 
 type Properties = {
   title: string;
-  id?: string;
   image?: ImageSourcePropType;
   iconName?: IconName;
   iconColor?: string;
-  onPress: (title: string, id?: string) => void;
+  onPress: () => void;
+  iconRight?: IconName;
+  onIconPress?: () => void;
 };
 
 const Card: React.FC<Properties> = ({
   title,
-  id = '',
   image = imagePlaceholder,
   iconName,
   iconColor,
   onPress,
+  iconRight,
+  onIconPress,
 }) => {
-  const handlePress = (): void => {
-    onPress(title, id);
+  const renderRightSwipeActions = (): React.ReactNode => {
+    return (
+      Boolean(iconRight) &&
+      iconRight && (
```

Comment:
@fshabanov i had this issue in cp-296, and  Boolean(iconRight) && is not work properly i had an error, so i wrote it like this

Reply:
@iamAlinaaa  What was the error? It doesn't really make sense to write `Boolean(iconRight) && iconRight` because you are checking one condition twice

Reply:
i just wrote iconRight, and @v0ldemar01 suggested me to write Boolean(iconRight) , but it gives an error because we need to check if iconName is not undefined
![Снимок экрана 2023-09-28 142701](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/df4b5b4f-c4b0-4a01-a698-ca3d429e5540)

Reply:
Did you try `name={iconRight as IconName}`?

Reply:
I did only iconRight && before , but I was said to change this , so I’ll use this name={iconRight as iconName}

Reply:
FYI you should always use boolean values in conditional rendering (especially in react native and especially with strings) because if we get an empty string for example then this condition will render empty string which is incorrect and will cause an error in react native. Why we should prefer `as iconName` instead of `&& iconName` - because we already checked that icon name exists, so this only ts issue that can be fixed using casting without unnecessary logic